snap/quota/quota.go

```diff
@@ -132,9 +133,13 @@ type Group struct {
 	// calculations
 	parentGroup *Group
 
-	// Snaps is the set of snaps that is part of this quota group. If this is
-	// empty then the underlying slice may not exist on the system.
+	// Snaps is the set of snaps that is part of this quota group. If both this
+	// and Services is empty then the underlying slice may not exist on the system.
 	Snaps []string `json:"snaps,omitempty"`
+
+	// Services is the set of snap services that is part of this quota group. If both
+	// this and Snaps is empty then the underlying slice may not exist on the system.
```

Comment:
this needs to explain what is the expected format for the content of Services, namely snap-name.app-name ?

Reply:
Good point!